R/utils.R

```diff
@@ -78,6 +78,15 @@ get_items <- function(label, subscales = c(), short_version = FALSE, configurati
 
       filtered_items <- as.data.frame(items[map(question_ids, function(x) grep(sprintf("TGMS_%04d", x), items$prompt_id)) %>% unlist() %>% unique(), ])
 
+      return(filtered_items[order(filtered_items$prompt_id), ])
+    } else if (label == "SMP") {
```

Comment:
You are in the wrong `if else `branch I think.

Reply:
Where else should it be? There are else if for GMS, SCS, SCA, too.  I can try a test.